apps/user/src/components/form/성적입력/VolunteerCalculator/VolunteerCalculator.tsx

```diff
@@ -0,0 +1,59 @@
+import { color, font } from '@maru/theme';
+import { NumberInput, Row, Td, Th } from '@maru/ui';
+import { styled } from 'styled-components';
+
+const VolunteerCalculator = () => {
+    return (
+        <Table>
+            <Row>
+                <Th width={162} height={56}>
+                    학년
+                </Th>
+                <Th width={654} height={56}>
+                    봉사시간
+                </Th>
+            </Row>
+            <Row>
+                <Td width={162} height={56} option="SECONDARY">
+                    1학년
+                </Td>
+                <Td width={654} height={56}>
+                    <NumberInput />
+                    <Paragraph>시간</Paragraph>
+                </Td>
+            </Row>
+            <Row>
+                <Td width={162} height={56} option="SECONDARY">
+                    1학년
+                </Td>
+                <Td width={654} height={56}>
+                    <NumberInput />
+                    <Paragraph>시간</Paragraph>
+                </Td>
+            </Row>
+            <Row>
+                <Td width={162} height={56} option="SECONDARY">
+                    1학년
+                </Td>
+                <Td width={654} height={56}>
+                    <NumberInput />
+                    <Paragraph>시간</Paragraph>
```

Comment:
```suggestion
                    <Hour>시간</Hour>
```

pargraph로 한 이유가 있나요.. 뜻이 불명확해서 잘 이해하기 힘드네요

Reply:
그냥 단순한 글자다보니 글이라는 단어와 연관된 영어를 찾다가 이렇게 했는데... Hour로 하는게 나을까요?

Reply:
네 더 나을것같아ㅛㅇ..

Reply:
알겠스비ㅣ낟..